src/eduid_userdb/actions/tou/user.py

```diff
@@ -46,7 +46,13 @@ def __init__(self, userid = None, tou = None, data = None):
         if data is None:
             data = dict(_id = userid,
                         tou = tou,
+                        #  We do not have a eppn here, but it is required for User.
+                        #  Since from the tou db we only update the central db with
+                        #  ToUs, this dummy eppn has no effect.
```

Comment:
Hmm, this worries me a bit. Haven't looked at how complicated that would be to do, but it seems it would be better to make this sub-class of User not require an eppn?